services/brig/brig.integration.yaml

```diff
@@ -34,7 +34,10 @@ aws:
   # ^ Comment this out if you don't want to journal user events
   prekeyTable: integration-brig-prekeys
   sqsEndpoint: http://localhost:4568 # https://sqs.eu-west-1.amazonaws.com
-  dynamoDBEndpoint: http://localhost:4567 # https://dynamodb.eu-west-1.amazonaws.com
+  # dynamoDBEndpoint: http://localhost:4567 # https://dynamodb.eu-west-1.amazonaws.com
```

Comment:
Are you sure you want to keep that line here as a comment if we want people to not do that any more?

Reply:
In my view we should do another pass (particularly with documentation) once we've convinced ourselves that all works well in staging.